doc/langdef.md

```diff
@@ -1177,9 +1288,20 @@ Regular expressions follow the [RE2 syntax](https://github.com/google/re2/wiki/S
       map indexing
     </td>
   </tr>
+  <tr>
+    <th rowspan="1">
+      @not_strictly_false
+    </th>
+    <td>
+      (bool) -> bool
+    </td>
+    <td>
+      false if argument is false, true otherwise (including errors and unknowns)
+    </td>
+  </tr>
   <tr>
     <th rowspan="2">
-      _in_
+      @in
```

Comment:
Same here - "@" function.

Reply:
I'll update the reference to just be `in`